recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
```

Comment:
You should add a comment to this cbc.yaml explaining why we target this macOS version. The comment should include, at a minimum:
* the name of the function(s)/symbol(s) that are needed;
* link(s) to the Apple documentation identifying what version of macOS the function/symbol was added;
* if it exists, a link to the upstream changelog entry saying this function(s)/symbol(s) is being used

Reply:
changes were made to reflect the request on the last commit

https://github.com/AnacondaRecipes/watchdog-feedstock/pull/1/commits/d46b6df18fd14dda23897545b6edd6243d1e9a2c

Reply:
I don't see any mention in the [watchdog readme](https://github.com/gorakhargosh/watchdog/blob/master/README.rst) about `kFSEventStreamEventFlagItemCloned` being used or any information about a minimum supported macOS version. If it's not actually mentioned, the link to the readme should be removed from the comment to avoid confusion.